.github/workflows/cdn-shared-publish.yml

```diff
@@ -60,13 +64,64 @@ jobs:
           role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCT_PROD }}:role/${{ inputs.GHA_ROLE }}
           aws-region: ${{ inputs.AWS_REGION }}
 
-      - name: Sync S3 content
+      - name: Sync custom domain CDN S3 content
+        # Only run this step if this is custom domain content (e.g., a folder at the root of bucket)
+        if: ${{ inputs.DOMAIN == 'custom' }}
         run: |
           if [ '${{ inputs.SYNC_PARAMS }}' != '' ]; then
             aws s3 sync . ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore" ${{ inputs.SYNC_PARAMS }}
           else
             aws s3 sync . ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore"
           fi
+          echo "Content is synchronized to ${{ inputs.S3URI }}" >> $GITHUB_STEP_SUMMARY
+
+      - name: Sync standard CDN S3 content
+        # Only run this step if this is standard content (e.g., a subfolder of the cdn/ folder)
+        if: ${{ inputs.DOMAIN == 'standard' }}
+        run: |
+          if [ '${{ inputs.SYNC_PARAMS }}' != '' ]; then
+            aws s3 sync ./$(echo ${{ inputs.S3URI }} | awk -F/ '{print $5}') ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore" ${{ inputs.SYNC_PARAMS }}
+          else
+            aws s3 sync ./$(echo ${{ inputs.S3URI }} | awk -F/ '{print $5}') ${{ inputs.S3URI }} --delete --exclude ".github/*" --exclude ".git/*" --exclude ".gitignore"
+          fi
+          echo "Content is synchronized to ${{ inputs.S3URI }}" >> $GITHUB_STEP_SUMMARY
 
       - name: Invalidate cache
-        run: aws cloudfront create-invalidation --distribution-id $(aws ssm get-parameter --name "/tfvars/libraries-website/cdnsite-id" --query 'Parameter.Value' --output text) --paths "/$(echo ${{ inputs.S3URI }} | awk -F/ '{print $4}')/*"
+        run: |
+          if [ '${{ inputs.DOMAIN }}' != 'standard' ]; then
```

Comment:
Inverting this if/else would make it slightly easier to reason about

Reply:
yup, good point. I had a reason for doing it this way, but since I don't remember that reason now, I'll clean this up (and the other ones too).